src/helpers/icons.js

```diff
@@ -46,7 +46,7 @@ export default [
     { name: 'Segment_Fixed', mode: '45', src: '/vendor/img/tool_icons/Mode_segmentfixed.svg' },
     { name: 'Angle_Fixed', mode: '46', src: '/vendor/img/tool_icons/Mode_anglefixed.svg' },
     { name: 'Locus', mode: '47', src: '/vendor/img/tool_icons/Mode_locus.svg' },
-    { name: 'Macro', mode: '48', src: '/vendor/img/tool_icons/Mode_tool.svg' },
+    // { name: 'Macro', mode: '48', src: '/vendor/img/tool_icons/Mode_tool.svg' },
```

Comment:
Please add annotation why they were removed from the array.

Reply:
Done!